### PR TITLE
Feature#18 checkin employee

### PR DESCRIPTION
**Pull request by OwenLobato**
# Description
☑︎ Fix DB schema for relations and foreign keys
☑︎ Modify controllers, views and models to insert relations
☑︎ Add clock in real time

### CODE REVIEW

File: app/assets/stylesheets/style_admin.css [hunk 1]
```diff
@@ -59,4 +59,12 @@ section{
...
\ No newline at end of file
+}
+
+.clock_input{
+background-color : Black; 
+color : White; 
+font-family : Verdana, Arial, Helvetica; 
+font-size : 20pt; 
+text-align : center;
+}
```
Comment (germanruzca) [inline app/assets/stylesheets/style_admin.css:70]: Check the indentation.

File: app/controllers/attendances_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  def show
+
+  end
+
+  def new
+
+  end
+
```
Comment (germanruzca) [inline app/controllers/attendances_controller.rb:14]: Empty methods could be in on lin
```suggestion
  def show; end
  
  def new; end
```

File: app/controllers/attendances_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  private
+
+  def attendance_params
+    params.require(:attendance).permit(:check_in, :check_out, :employee_id)
+  end
+
+
+end
```
Comment (germanruzca) [inline app/controllers/attendances_controller.rb:31]: Blank line at the end here

File: app/models/attendance.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class Attendance < ApplicationRecord
+    belongs_to :employee
+end
```
Comment (germanruzca) [inline app/models/attendance.rb:3]: Blank line at the end here

File: app/views/admin/index.html.erb [hunk 5]
```diff
@@ -14,3 +14,4 @@
     <% end %>
   </div>
 </div>
+<%= link_to  "Volver", '/' %>
```
Comment (germanruzca) [inline app/views/admin/index.html.erb:17]: As suggestion, you can use the path helper `root_path`

**Review by germanruzca — CHANGES_REQUESTED** [review]

**Review by AgustinPalmaM — DISMISSED** [review]
LGTM

File: app/assets/stylesheets/style_admin.css (hunk 1 above)
Comment (OwenLobato) [reply app/assets/stylesheets/style_admin.css:70]: Ok

File: app/controllers/attendances_controller.rb (hunk 2 above)
Comment (OwenLobato) [reply app/controllers/attendances_controller.rb]: Ok

File: app/models/attendance.rb (hunk 4 above)
Comment (juperameza) [reply app/models/attendance.rb:3]: Agree with German 
You should activate this [setting](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files) on vs code

File: app/javascript/clock.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  if (str_segundo.length == 1)
+     segundo = "0" + segundo
+
+  str_minuto = new String (minuto)
+  if (str_minuto.length == 1)
+     minuto = "0" + minuto
+
+  str_hora = new String (hora)
+  if (str_hora.length == 1)
+     hora = "0" + hora
```
Comment (juperameza) [inline app/javascript/clock.js:17]: As a suggestion you can use [Js template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

**Review by juperameza — CHANGES_REQUESTED** [review]
Fix @germanruzca revisions

**Review by germanruzca — APPROVED** [review]
LGTM 🚀

**Review by juperameza — APPROVED** [review]
LGTM!

**Review by AgustinPalmaM — APPROVED** [review]
lgtm

**Review by V-Juarez — APPROVED** [review]

**Pull request merged**